src/appleseed/renderer/kernel/rendering/pixelrendererbase.h

```diff
@@ -32,20 +32,26 @@
 
 // appleseed.renderer headers.
 #include "renderer/kernel/rendering/ipixelrenderer.h"
+#include "renderer/modeling/frame/frame.h"
 
 // appleseed.foundation headers.
+#include "foundation/image/tile.h"
```

Comment:
I presume you need this for `std::unique_ptr<>`? If you do, then please remove the `Tile` forward declaration below.